src/font/lv_font_loader.c

```diff
@@ -335,6 +354,82 @@ static int32_t load_cmaps(lv_fs_file_t * fp, lv_font_fmt_txt_dsc_t * font_dsc, u
     return success ? cmaps_length : -1;
 }
 
+#if LV_USE_FONT_DYNAMIC_LOAD
+static uint8_t * load_glyph_bitmap(void * fmt_dsc, void * glyph_dsc)
+{
+
+    lv_font_fmt_txt_glyph_dsc_t * gdsc = (lv_font_fmt_txt_glyph_dsc_t *)glyph_dsc;
+
+    lv_font_fmt_txt_dsc_t * fdsc = (lv_font_fmt_txt_dsc_t *)fmt_dsc;
+    lv_font_fmt_txt_glyph_loader_t * bitmap_loader = fdsc->loader;
+
+    /*header*/
+    int32_t header_length = read_label(bitmap_loader->fp, 0, "head");
+    if(header_length < 0) {
+        return NULL;
```

Comment:
Please add a LOG_ERR telling what is invalid

Reply:
Thanks, updated!